lib/fastlane/plugin/appcenter/helper/appcenter_helper.rb

```diff
@@ -48,6 +48,8 @@ def self.create_release_upload(api_token, owner_name, app_name)
         when 404
           UI.error("Not found, invalid owner or application name")
           false
+        when 500...600
```

Comment:
Why don't we just stop for other error codes? For example when it's 401, shouldn't we also error and consider the command failed?

Reply:
I think that makes sense - in terms of versioning, this would be a breaking change to existing functionality, so I think we might need to raise the minor version then.